src/mgmt/osp-sah.ks

```diff
@@ -595,15 +591,11 @@ fi
 
 subscription-manager repos ${ProxyInfo} '--disable=*' --enable=rhel-7-server-rpms --enable=rhel-7-server-optional-rpms
```

Comment:
Change the repos to --enable=rhel-8-for-x86_64-baseos-rpms --enable=rhel-8-for-x86_64-appstream-rpms